### PR TITLE
autoid: increase step of allocator.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: meta/autoid/autoid.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	step = 1000
+	step = 5000
```
Comment (shenli) [inline meta/autoid/autoid.go]: Why change it to 5k?

Comment (ngaut) [inline meta/autoid/autoid.go]: For batch allocation.

**Comment by shenli** [comment]
LGTM

**Comment by hanfei19910905** [comment]
LGTM

**Pull request merged**